CentumFramework/GoogleSheets/Entities/Spreadsheet.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace CentumFramework.GoogleSheets.Entities
 {
-    public class Spreadsheet : IEnumerable<Sheet>
+    public class Spreadsheet : IReadOnlyCollection<Sheet>
```

Comment:
```suggestion
    public class Spreadsheet : IReadOnlyCollection<Sheet>, ICollection<Sheet>
```

Я не помню почему, но ридонли версия не реализует обычную, а она часто бывает полезной

Reply:
Тут наверное такое не понадобится, так как все изменения будут делаться через отдельные методы

Reply:
Чет я кстати вообще перепутал
Очев же почему ридонли версия не имплементит обычную))
Чет показалось, что наоборот ICollection уже есть и надо ридонли